### PR TITLE
add deadlock error resolution

### PR DESCRIPTION
**Pull request by voroyam**
added a explanation + solution with guides on how to resolve this error

### CODE REVIEW

File: modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc [hunk 1]
```diff
@@ -438,3 +438,22 @@ PostgreSQL: \c owncloud; \d
 MySQL     : quit
 PostgreSQL: \q
 ....
+
+[[how-to-solve-deadlock-error]]
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc]: This explicit anchor/id isn't required, as one exactly the same will be auto-generated.

File: modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc [hunk 2]
```diff
@@ -438,3 +438,22 @@ PostgreSQL: \c owncloud; \d
...
+=== How to solve deadlock error
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc]: Please use [title case](https://www.snappysentences.com/sentence-case-v-title-case/) for headers.

Comment (voroyam) [reply modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc]: why not just write "replace this with this"? 

If you have a question for me I don't tell you "Yeah mate, just google this and then you get a page, read that page, somewhere on that page you get the answer I could tell you right now"

**Comment by voroyam** [comment]
Although I don't agree with the change from the warning style of section to a normal code block, I understand that you are the authority on the formatting and language part of this. So, thanks. Mergin.

**Comment by mmattel** [comment]
@voroyam pls set the PR labels including backporting or not...

**Pull request merged**